packages/Dialog/src/dialog.scss

```diff
@@ -0,0 +1,90 @@
+@use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
+@use '~@igloo-ui/tokens/dist/fonts';
+
+:root {
+  /* Default */
+  --ids-dialog-font-family: #{tokens.$primary-font-family};
+  --ids-dialog-font-size: #{tokens.$font-size-4};
+  --ids-dialog-max-width: calc(100% - #{tokens.$space-5});
+  --ids-dialog-radius: #{tokens.$border-radius-sm};
+  --ids-dialog-header-direction: column-reverse;
+  --ids-dialog-header-height: auto;
+  --ids-dialog-header-font-size: #{tokens.$font-size-6};
+  --ids-dialog-header-padding: #{tokens.$space-3} #{tokens.$space-3} #{tokens.$space-2};
+  --ids-dialog-content-padding: #{tokens.$space-2} #{tokens.$space-4} #{tokens.$space-4};
+  --ids-dialog-sub-title-line-height: #{tokens.$line-height-xxxl};
+  --ids-dialog-sub-title-color: #{tokens.$grey-600};
+  --ids-dialog-close-margin: #{tokens.$space-0};
+  --ids-dialog-close-align: flex-end;
+  --ids-dialog-buttons-margin: #{tokens.$space-5} -#{tokens.$space-0} -#{tokens.$space-2};
+  --ids-dialog-button-margin: #{tokens.$space-2} #{tokens.$space-0};
+}
+
+@media (min-width: #{tokens.$breakpoints-sm}) {
+  :root {
+    --ids-dialog-content-padding: #{tokens.$space-0} #{tokens.$space-5} #{tokens.$space-5};
+    --ids-dialog-buttons-margin: #{tokens.$space-7} -#{tokens.$space-2} #{tokens.$space-0};
+    --ids-dialog-button-margin: 0 #{tokens.$space-2};
+  }
+}
+
+.ids-modal.ids-dialog {
+  border-radius: var(--ids-dialog-radius);
```

Comment:
why redefine these values (radius, font-family, font-size)? From what I see it remains identical to the modal.